src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,146 @@
+<!---{
+    "experiment": true,
+    "component": "amp-live-list"
+  }--->
+<!--
+  #### Introduction
+
+  The [amp-live-list](https://github.com/ampproject/amphtml/blob/f2aed679a4c0611ea077985cbe0d1926c7d0a6f5/extensions/amp-live-list/amp-live-list.md) component add support for publishing content that updates live in the AMP page as new content is available in the source document.
+
+  This example shows an implementation with a server rendering a series of 10 elements, `amp-live-list` will poll the page every 15 seconds. Every time `amp-live-list` polls the page, the server adds a new item to the page. Refer to [server.go](https://github.com/ampproject/amp-by-example/blob/master/server.go) for implementation details.
+
+  The sample live blog stops after ten articles. If you want to start over, delete cookies from ampbyexample.com or open this page again in an incognito/private tab.
+
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+  Import the `amp-live-list` component in the header.
+
+  Use `amp-live-list-item` to style the children of the `amp-live-list`. Refer to the [documentation](https://github.com/ampproject/amphtml/blob/f2981e72843c8e71a9cd2535282e4c4947bb6882/extensions/amp-live-list/amp-live-list.md) for details.
```

Comment:
Right, I was not using that class anymore but I could in order to center the cards.
Look at [this](https://www.dropbox.com/s/6bikotoh0xa4la2/am-live-list-center.png?dl=0)
I realized that by doing:
`.amp-live-list-item {
      justify-content: center;  
      display: flex;     
    }
    .blog{
      padding: 0;
      min-width: 70%;
      background: #fff;
    }
    .blog p {
      padding: 8px;
      margin: 0;
    }`
What do you think?